Classes/Typo3PageIndexer.php

```diff
@@ -315,7 +315,10 @@ protected function getPageDocument()
         $document->setField('rootline', $rootline);
 
         // access
-        $document->setField('access', (string)$this->pageAccessRootline);
+        $access = (string)$this->pageAccessRootline;
+        if (trim($access) !== "") {
```

Comment:
I know, but then there there is a bug somewhere else because a PageAccessRootline generated with Rootline::getAccessRootlineByPageId (casted to string in Typo3PageIndexer::getPageDocument) produces an empty access string. When this gets past to solr it will we "", when passing nothing the document will be created with "c:0" which is the default from the schema. Therefore i think the check makes sence.